logging/loggingtest/logger.go

```diff
@@ -96,8 +97,7 @@ func New() *Logger {
 	mute := make(chan bool)
 	quit := make(chan struct{})
 
-	// start muted to reduce spam during failing tests
```

Comment:
I think rich log output is exactly what is needed to debug the failing test. This PR proposes to unmute when verbose test output is enabled which requires re-running failing test with `go test -v`. Maybe we should simply unmute testlogger unconditionally.